L1Trigger/L1THGCal/interface/backend/HGCalMulticlusteringHistoImpl.h

```diff
@@ -71,6 +71,7 @@ class HGCalMulticlusteringHistoImpl{
     std::vector<unsigned> binsSumsHisto_;
     double histoThreshold_ = 20.;
     std::vector<double> neighbour_weights_;
+    unsigned neighbour_weights_size_ = 9;
```

Comment:
Since it is not meant to be dynamically changed, it should be declared with `static contexpr unsigned` like the `kROverZMin_` variable below